x-pack/legacy/plugins/maps/public/layers/sources/es_pew_pew_source/es_pew_pew_source.js

```diff
@@ -126,22 +125,22 @@ export class ESPewPewSource extends AbstractESAggSource {
   createDefaultLayer(options) {
     const styleDescriptor = VectorStyle.createDescriptor({
       [vectorStyles.LINE_COLOR]: {
-        type: VectorStyle.STYLE_TYPE.DYNAMIC,
+        type: DynamicStyleProperty.type,
         options: {
           field: {
-            label: COUNT_PROP_LABEL,
-            name: COUNT_PROP_NAME,
+            label: AbstractESAggSource.COUNT_PROP_LABEL,
```

Comment:
AbstractESAggSource does not have properties COUNT_PROP_LABEL or COUNT_PROP_NANE (and COUNT_PROP_NANE is a typo, NANE). Should just import directly from constants.